webapp/store/logic.py

```diff
@@ -285,3 +285,22 @@ def get_confinement(channel_map, track, risk):
                     return release["confinement"]
 
     return None
+
+
+def get_version(channel_map, track, risk):
+    """Get the version for a channel
+
+    :param channel_map: Channel map list
+    :param track: The track of the channel
+    :param risk: The risk of the channel
+
+    :returns: The version
+    """
+    for arch in channel_map:
+        if arch in channel_map and track in channel_map[arch]:
```

Comment:
Isn't `if arch in channel_map` always true because you are iterating over `for arch in channel_map`?

Reply:
👍 just updated

Reply:
@tbille mentioned that these two functions to get confinement and get version can do with some refactoring